test/test_jit.py

```diff
@@ -15618,6 +15618,11 @@ def add_autograd_test(
     if 'complex' in variant_name or name in ['view_as_complex', 'complex']:
         return
 
+    # Disable tests for lu from common_methods_invocations.py
```

Comment:
Do you mind setting up a setting up a list and doing it in the same way as `EXCLUDE_ALIAS` ?

Reply:
There is an inclusion test `name in EXCLUDE_ALIAS`, so I extended that list by `lu`. But it kills all the `lu` tests, however, these tests are the only ones generated in this PR.